### DISCUSSION TITLE
Unable to reverify GitHub Education

### DISCUSSION THREAD

**Discussion by AneudysOrtiz** [community/community#58484] [GitHub Education]
### Hi everyone,

I don't have access to my GitHub education package but I'm unable to reverify since it says I am already verified.

<img width="671" alt="image" src="https://github.com/community/community/assets/25928891/7a0d5fb3-4b6a-4d74-9f41-4c9ba4187578">
<img width="658" alt="Screenshot 2023-06-19 at 1 34 33 PM" src="https://github.com/community/community/assets/25928891/59bd450f-ca11-45b1-b2dd-278e76351cb5">

**Comment by mrepol742** [comment]
You need to wait atleast 3 days or more for the academic benefits after they approved your github edu accnt.

**Reply by AneudysOrtiz** [reply]
But they were already approved 1 year ago as you can see in the first picture

**Reply by mrepol742** [reply]
![image](https://github.com/community/community/assets/62317165/1e0e2053-62de-4b5b-a447-0d7ec1acce67)
You are already using github student edu package. Look at your profile you have github pro.

Check this link:
https://education.github.com/pack/offers

**Reply by AneudysOrtiz** [reply]
You're right! 

I was using GitKraken and it stopped working. After login again, it worked
